packages/eslint-config-adidas-es5/CHANGELOG.md

```diff
@@ -1,3 +1,78 @@
+# 1.3.0
+
+- Updated ESLint to version 7.
+- Removed deprecated rules that were moved to `eslint-plugin-node`.
+- Added all missing rules for ES6, 7, 8... as `off`:
```

Comment:
@moelders if you meant this, I've added to ES5 configuration all missing ES6+ rules just to be sure their defaults don't get applied incorrectly for ES5 codebases!